integreat_cms/summ_ai_api/summ_ai_api_client.py

```diff
@@ -80,20 +98,28 @@ async def translate_text_field(self, session, text_field):
                 },
             ) as response:
                 # Wait for the response
-                response_data = await response.json()
-                # Check whether the text was translated successfully
-                if "translated_text" not in response_data:
-                    if "error" in response_data:
-                        raise SummAiException(
-                            f"API error: {response.status} - {response_data['error']}"
-                        )
-                    raise SummAiException(
-                        f"Unexpected API result: {response.status} - {response_data!r}"
+                try:
+                    response_data = await response.json()
+                except (
+                    aiohttp.ContentTypeError,
+                    aiohttp.web_exceptions.HTTPBadRequest,
```

Comment:
it produces AttributeError at /augsburg/pages/de-si/machine-translate/
`module 'aiohttp' has no attribute 'web'`